backend/DynamicDispatch.cpp

```diff
@@ -251,8 +267,8 @@ Value *CodeGenerator::callDynamicFun(const Node &node, Value *rtFnPointer, const
   Value *packedArgSignature = ConstantInt::get(*TheContext, APInt(64, 0, false));
 
   vector<TypedValue> pointerCallArgs;
-
-  if(args.size() > 20) throw CodeGenerationException("More than 20 args not yet supported", node); 
+  /* 21 because varaidic adds one arg that packs the remaining */
+  if(args.size() > 21) throw CodeGenerationException("More than 20 args not supported", node); 
```

Comment:
Czy ten warunek nie powinien być
`(funkcja wariadyczna i args.size() > 21) lub (funkcja niewariadyczna i args.size() > 20)`?
Frontend na razie i tak nie przepuści funkcji z dokładnie 21 parametrami (konkretnie `eval` w `clojure.rt.core/analyze`), więc nie jestem pewien.

Reply:
Probklem polega na tym że na tym etapie nie wiemy która to jest metoda, bo ten obiekt jest w runtime.